# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
bool unsafe
bool timeout
